beesdoo_website_pos_order/controllers/main.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015-2016 Odoo S.A.
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# Copyright 2017-2018 Rémy Taymans <remytaymans@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from werkzeug.exceptions import Forbidden, NotFound
+
+from openerp import http
+from openerp.exceptions import AccessError, MissingError
+from openerp.http import request
+
+from openerp.addons.website_portal_v10.controllers.main import WebsiteAccount
+
+
+class CooperatorWebsiteAccount(WebsiteAccount):
+
+    @http.route()
+    def account(self):
+        """ Add POS Order to main account page """
+        response = super(CooperatorWebsiteAccount, self).account()
+        partner = request.env.user.partner_id
+
+        pos_order_mgr = request.env['pos.order'].sudo()
```

Comment:
sudo() should not come here. always instantiate it on the function call. here you put a sudo() wildcard on all the function calls which is maybe not recommended